src/netlog/netlog-tls.h

```diff
@@ -14,7 +14,6 @@ typedef enum OpenSSLCertificateAuthMode {
         OPEN_SSL_CERTIFICATE_AUTH_MODE_DENY     = 1 << 2,
         OPEN_SSL_CERTIFICATE_AUTH_MODE_WARN     = 1 << 3,
         OPEN_SSL_CERTIFICATE_AUTH_MODE_MAX      = 1 << 4,
-        OPEN_SSL_CERTIFICATE_AUTH_MODE_INVALID  = -1,
 } OpenSSLCertificateAuthMode;
```

Comment:
It's used by the parser. If we give anything which is not expected parser will set it to invalid. Hence we require this in the enum.

Reply:
In my testing it doesn't:

The default value gets set via
https://github.com/systemd/systemd-netlogd/blob/085384d53b9fdbb98aa860eb88c06bba8ef125da/src/netlog/systemd-netlogd.c#L172
at
https://github.com/systemd/systemd-netlogd/blob/085384d53b9fdbb98aa860eb88c06bba8ef125da/src/netlog/netlog-manager.c#L637.

And afterwards the configuration file is parsed at
https://github.com/systemd/systemd-netlogd/blob/085384d53b9fdbb98aa860eb88c06bba8ef125da/src/netlog/systemd-netlogd.c#L178
and if `TLSCertificateAuthMode=` is not specified the value does not change and on an invalid setting, e.g. the empty `TLSCertificateAuthMode=`, the parser will issue `[/etc/systemd/netlogd.conf:4] Failed to parse 'TLSCertificateAuthMode=', ignoring.`.